main.md

```diff
@@ -40,22 +40,22 @@ A mesh topology combined with a novel consensus PoST protocol that allows anyone
 
-> To deep-dive into the technical details of the Spacemesh protocol, read the [Spacemesh Protocol paper](https://spacemesh.io/spacemesh-protocol-v1-0).
+> To deep-dive into the technical details of the Spacemesh protocol, read the [Spacemesh protocol paper](https://spacemesh.io/spacemesh-protocol-v1-0).
 
-## What is the Spacemesh Testnet?
-The Spacemesh Testnet is a public beta release of the Spacemesh p2p software platform. Anyone with a computer and an Internet connection can join without any permission from anywhere and participate in the Spacemesh p2p network.
+## What is Spacemesh Testnet?
+Spacemesh Testnet is a public beta release of the Spacemesh p2p software platform. Anyone with a computer and an Internet connection can join without any permission from anywhere and participate in the Spacemesh p2p network.
 
-The Testnet is designed to test the Spacemesh protocols and software prior to release of the Spacemesh Mainnet and the Spacemesh cryptocurrency. Note that the cryptocurrency enabled by the Testnet is ``monopoly money'' without any real world value.
+Spacemesh Testnet is designed to test the Spacemesh protocol and software prior to release of Spacemesh Mainnet and the Spacemesh cryptocurrency. Note that the cryptocurrency enabled by the Testnet is ``monopoly money'' without any real world value.
 
 ## What is Spacemesh 0.1 ?
-Spacemesh 0.1 is the name of the first release of the Testnet. Everything described in this guide is about this release. We are releasing capabilities that we believe are ready for testing. We have much more in the pipeline and we plan to release updates to the Testnet throughout 2020. This guide will be updated as we release new Testnet features.
+Spacemesh 0.1 is the first release of the Testnet. Everything described in this guide is about this release. We are releasing capabilities that we believe are ready for testing. We have much more in the pipeline and we plan to release updates to the Testnet throughout 2020. This guide will be updated as we release new Testnet features.
 
 ## Why should I join the Spacemesh Testnet?
-Joining the Testnet helps Spacemesh launch a high-quality Mainnet and the `Spacemesh Coin` (SMC) cryptocurrency. Spacemesh is a decentralized ledger designed to be formed from a large number of home desktop PC owners who run the Spacemesh software on their PCs. To test such a network we need a large number of block producers to join the Testnet in addition to the close Testnets which we have been running.
+Joining the Testnet helps Spacemesh launch a high-quality Mainnet and the `Spacemesh Coin` (SMC) cryptocurrency. Spacemesh is a decentralized ledger designed to be formed from a large number of home desktop PC owners who run the Spacemesh software on their PCs. To test such a network we need a large number of block producers to join the Testnet in addition to the close Testnets which we have been running. 
```

Comment:
I don't see any value in stating that we have close testnets, and would remove this part of the sentence.